bee-storage/bee-storage-test/src/message_id_to_metadata.rs

```diff
@@ -54,9 +54,18 @@ pub fn message_id_to_metadata_access<B: StorageBackend>(storage: &B) {
     assert_eq!(results.len(), 1);
     assert!(matches!(results.get(0), Some(Ok(None))));
 
-    Insert::<MessageId, MessageMetadata>::insert(storage, &message_id, &metadata).unwrap();
-
+    InsertStrict::<MessageId, MessageMetadata>::insert_strict(storage, &message_id, &metadata).unwrap();
     assert!(Exist::<MessageId, MessageMetadata>::exist(storage, &message_id).unwrap());
+
+    {
+        let index = metadata.milestone_index().map_or(0, |i| *i + 1);
+        let mut metadata = metadata.clone();
+        metadata.set_milestone_index(MilestoneIndex(index));
```

Comment:
What's the point of this ?

Reply:
calling `insert_strict` with a different metadata and checking that the old metadata was not overwritten. I added some comments explaining it

Reply:
But you end up calling `insert_strict` with a random metadata

Reply:
lol, smooth brain time. I'll fix it